### PR TITLE
Fixes the TGUI Error Logo

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
This fixes the logo for the TGUI error screen.
## Why It's Good For The Game
This allows for the logo to be rendered as intended.
## Images of changes
5MZOHjBQ4n
## Testing
Broke a game UI.
Observed error screen for proper formatting.
## Changelog
:cl:
fix: Fixed logo for tgui error screen
/:cl:

### CODE REVIEW

**Review by DGamerL — APPROVED** [review]
If no TGUI rebuild is needed, this LGTM

File: tgui/packages/tgui/public/tgui.html [hunk 1]
```diff
@@ -133,9 +133,12 @@
...
-        ███╗ ██╗████████╗ ██████╗ ███████╗ ████╗ ██║╚══██╔══╝██╔═══██╗██╔════╝
-        ██╔██╗ ██║ ██║ ██║ ██║███████╗ ██║╚██╗██║ ██║ ██║ ██║╚════██║ ██║ ╚████║
-        ██║ ╚██████╔╝███████║ ╚═╝ ╚═══╝ ╚═╝ ╚═════╝ ╚══════╝
+        ███╗   ██╗████████╗ ██████╗ ███████╗
+        ████╗  ██║╚══██╔══╝██╔═══██╗██╔════╝
+        ██╔██╗ ██║   ██║   ██║   ██║███████╗
+        ██║╚██╗██║   ██║   ██║   ██║╚════██║
+        ██║ ╚████║   ██║   ╚██████╔╝███████║
+        ╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚══════╝
       </div>
```
Comment (Contrabang) [inline tgui/packages/tgui/public/tgui.html:142]: Can you make sure this works normally after prettier is run?

Comment (Burzah) [reply tgui/packages/tgui/public/tgui.html:142]: Ran prettier on this, the file remained unchanged. Good call!

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**